src/App.js

```diff
@@ -1,19 +1,158 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
 
+import todos from './api/todos';
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+import TodoList from './components/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
```

Comment:
you can get userName from User object and add it to the todo, as you don't need other properties of the user.